README.md

```diff
@@ -29,10 +29,15 @@ You will need the following things properly installed on your computer.
 
 Make use of the many generators for code, try `ember help generate` for more details
 
-### Running Tests
+### Running Tests and generating coverage
 
-* `ember test`
-* `ember test --server`
+First of all, you need karme-cli to do this. Make sure to install it globaly:
+
+* `npm install -g karma-cli`
```

Comment:
Why install the package globally? I think it should be installed only for the project. Otherwise we will sooner or later have problems with other projects requiring the package in a different version.